src/TesApi.Web/Storage/DefaultStorageAccessProvider.cs

```diff
@@ -85,7 +85,7 @@ public override async Task<bool> IsPublicHttpUrlAsync(string uriString)
         }
 
         /// <inheritdoc />
-        public override async Task<string> MapLocalPathToSasUrlAsync(string path, bool getContainerSas = false)
+        public override async Task<string> MapLocalPathToSasUrlAsync(string path, bool getContainerSas = false, SharedAccessBlobPermissions permissions = SharedAccessBlobPermissions.Read | SharedAccessBlobPermissions.Create | SharedAccessBlobPermissions.Write | SharedAccessBlobPermissions.List)
```

Comment:
Given the fact that `permissions` are, for all intents and purposes, always provided, while `getContainerSas` is not, I would recommend swapping the order of those two (and please consider making `permissions` required rather then optional).